cassandra/cqlengine/statements.py

```diff
@@ -550,6 +550,7 @@ class SelectStatement(BaseCQLStatement):
     def __init__(self,
                  table,
                  fields=None,
+                 distinct_fields=[],
```

Comment:
Curious about why this was added in the middle of the args. I think this is probably okay as an internal API, but I usually only append keyword arguments where possible.